modules/nw-ovn-k-anp-banp-status.adoc

```diff
@@ -0,0 +1,145 @@
+//module included in the following assemblies:
+//
+//
+:_mod-docs-content-type: PROCEDURE
+[id="checking-anp-banp"]
+== Checking `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy`
```

Comment:
I'd prefer to move this just like https://76667--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/network_security/network_policy/viewing-network-policy => the aim here is exactly like how its mentioned in viewing an ANP ; but understanding how the status can help kind of thing.

So more than troubleshooting perhaps viewing an ANP is a better thing for consistency?

Reply:
Also like that "	
If you log in to the web console with cluster-admin privileges, you have a choice of viewing a network policy in any namespace in the cluster directly in YAML or from a form in the web console." bit => would be good to check and see if that's doable

Reply:
I agree about all the Netpol stuff. For GA, moving this to Troubleshooting. Ok, compromise?

Reply:
Post GA, I agree. We will put it there. For now though, ok to move it to troubleshooting as part of `Checking ANP and BANP`? @tssurya cc @asood-rh